### PR TITLE
Nc news api 6

### PR DESCRIPTION
**Pull request by danny-817**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -91,3 +91,52 @@ describe("/api", () => {
...
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.comments.length).toBe(2);
+          body.comments.forEach((comment) => {
+            expect(comment).toHaveProperty("votes");
+            expect(comment).toHaveProperty("comment_id");
+            expect(comment).toHaveProperty("body");
+            expect(comment).toHaveProperty("created_at");
+            expect(comment).toHaveProperty("author");
+            expect(comment).toHaveProperty("article_id");
```
Comment (Slyose) [inline __tests__/app.test.js:111]: We should also check the keys, either their value (if you know exactly what it will be, life for articleID) or their data type. We can achieve this like so:

`expect(comment).toHaveProperty("votes", expect.any("Number"));`


.toHaveProperty can take another argument for the value you expect the key to have, wich helps us make our tests even stronger.

**Review by Dutton1990 — CHANGES_REQUESTED** [review]
Nearly there with this one!

You also need a test for an article that has no comments (i think article 2 has none) 

This should return 200 and an empty array, it's not an error there's just no comments yet. 

To do this you'll have to check that the article exists, otherwise you'll run into returning an error.

**Review by Slyose — COMMENTED** [review]
Looking good so far! Left a comment to help you make even better tests. Will be good to merge after that!
